src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form"
+    data-qa="big"
+    method="post">
+      <input
+      class="form__big"
+      data-qa="keypress"
+      type="text"
+      placeholder="Try ”Los Angeles”"
+      >
+    </form>
+
+    <form
+    class="form"
+    data-qa="small"
+    method="post">
+      <input
+      class="form__small"
```

Comment:
for small input set modifier which will be reset only styles which different from big input
class="form__field form__field--small"